### PR TITLE
Chunter last views

### PR DESCRIPTION
**Pull request by BykhovDenis**
Signed-off-by: Denis Bykhov [80476319+BykhovDenis@users.noreply.github.com](mailto:80476319+BykhovDenis@users.noreply.github.com)

### CODE REVIEW

**Comment by ksilobait** [comment]
Hovering over the first item leads to partial overlapping of reply count
image

**Comment by ksilobait** [comment]
Switching between sections of the app (e.g. Chat -> Contacts -> Chat) closes the channel but doesn't close the thread, is this expected?

File: plugins/chunter-resources/src/components/Message.svelte [hunk 1]
```diff
@@ -14,10 +14,13 @@
 -->
 
 <script lang="ts">
+import { Attachment } from '@anticrm/attachment';
```
Comment (haiodo) [inline plugins/chunter-resources/src/components/Message.svelte]:
```suggestion
  import { Attachment } from '@anticrm/attachment'
```

File: plugins/chunter-resources/src/components/Message.svelte [hunk 2]
```diff
@@ -14,10 +14,13 @@
...
+
   import { AttachmentDocList } from '@anticrm/attachment-resources'
+import AttachmentList from '@anticrm/attachment-resources/src/components/AttachmentList.svelte';
```
Comment (haiodo) [inline plugins/chunter-resources/src/components/Message.svelte]:
```suggestion
  import AttachmentList from '@anticrm/attachment-resources/src/components/AttachmentList.svelte'
```

**Review by haiodo — APPROVED** [review]

**Comment by BykhovDenis** [comment]
> Switching between sections of the app (e.g. Chat -> Contacts -> Chat) closes the channel but doesn't close the thread, is this expected?

Currently is expected, it can be changed in future

**Review by haiodo — APPROVED** [review]

**Comment by ksilobait** [comment]
I see these `x replies Last reply` only in dev build but not in docker build, is this expected?
image

**Comment by BykhovDenis** [comment]
> I see these `x replies Last reply` only in dev build but not in docker build, is this expected? image

You should rebuild docker and update workspaces

**Comment by ksilobait** [comment]
> > I see these `x replies Last reply` only in dev build but not in docker build, is this expected? image
> 
> You should rebuild docker and update workspaces

Ok, now it works for all new messages

**Pull request merged**